tests/test_discovery.py

```diff
@@ -515,6 +515,25 @@ def test_building_with_developer_key_skips_adc(self):
         # application default credentials were used.
         self.assertNotIsInstance(plus._http, google_auth_httplib2.AuthorizedHttp)
 
+    def test_api_endpoint_override_from_client_options(self):
+        discovery = open(datafile("plus.json")).read()
+        api_endpoint = "https://foo.googleapis.com/"
+        options = google.api_core.client_options.ClientOptions(
+            api_endpoint=api_endpoint
+        )
+        plus = build_from_document(discovery, client_options=options)
+
+        self.assertEquals(plus._baseUrl, api_endpoint)
```

Comment:
(did not check the rest of the PR)

`assertEquals()` is deprecated, better to use `assertEqual()` instead.

Reply:
Done. Grepped through the files and found some other instances so I changed them en masse. Hopefully it doesn't make the diff to hard to read.

Reply:
The diff is fine if going through on a per-commit basis, thanks!